extensions/datetime-range/examples/example-video.json

```diff
@@ -1,7 +1,7 @@
 {
-  "stac_version": "0.8.0",
+  "stac_version": "0.8.1",
   "stac_extensions": [
-    "datetime-range"
+    "dtr"
```

Comment:
This is wrong, the stac_extensions are the folder names an that's still datetime-range.

Reply:
fixed